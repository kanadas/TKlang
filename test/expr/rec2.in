let x = if x < 10 then 2 * x else 1 in x
