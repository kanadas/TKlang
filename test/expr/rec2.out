expr/rec2.in
